packages/models/package.json

```diff
@@ -8,6 +8,9 @@
   "exports": {
     ".": "./dist/index.js"
   },
+  "config":  { 
+    "protocVersion": "26.1"
+  },
```

Comment:
I think this is not needed here

Reply:
is it confirmed that moving this config here is ok?